packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -44,14 +52,34 @@ export interface ImageInputProps
    * An accessible label for the "clear" icon button.
    */
   clearButtonLabel: string;
+  /**
+   * A callback function to call when the user has selected an image.
+   */
+  onChange: (event: File | null) => Promise<void>;
```

Comment:
When would the `event` be `null`?

Reply:
The FileList would be empty when the input is cleared, but the `onChange` event isn't called on clear, right?

Reply:
Fixed in 52f3f0f

Reply:
Did you see my question above? Are there any downsides to not calling `onChange` when the FileList is empty? Maybe browser inconsistencies?

Reply:
Oh no, I missed it! No, there are no downsides, onChange should not be called when a file is cleared 🙂 

What do you mean about browser inconsistencies? As in, would some browsers still need to rely on the onChange? I think it should be fine, but anyways we'll need to manually test the behavior on the major browsers we support 👍

Reply:
I mean do browsers consistently call/not call the `onChange` event when clearing the input?

Reply:
I can check FF and Safari now, Chrome doesn't call it. Even if they do, we return before calling the onChange if the file is falsy.